py4DSTEM/process/diffraction/crystal_ACOM.py

```diff
@@ -27,8 +27,11 @@ def orientation_plan(
     angle_step_in_plane: float = 2.0,
     accel_voltage: float = 300e3,
     corr_kernel_size: float = 0.08,
-    radial_power: float = 1.0,
-    intensity_power: float = 0.25,  # New default intensity power scaling
+    sigma_excitation_error: float = 0.02,
+    precession_angle_degrees=None,
+    power_radial: float = 1.0,
+    power_intensity: float = 0.25,
+    power_intensity_experiment: float = 0.25,
```

Comment:
Precession steps should probably be able to be set here as well

Reply:
Same comment as previous.